### PR TITLE
release-tests: install libasan as native dependency

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
See #18099 and https://github.com/RIOT-OS/RIOT/runs/6534330826?check_suite_focus=true

### Testing procedure
Will start a standalone release test.

### Issues/PRs references
Follow-up on #18099.

### CODE REVIEW

**Review by kaspar030 — APPROVED** [review]
ACK.

File: .github/workflows/release-test.yml [hunk 1]
```diff
@@ -116,6 +116,11 @@ jobs:
       if: ${{ matrix.pytest_mark == 'not iotlab_creds' }}
       run: |
         sudo RIOT/dist/tools/tapsetup/tapsetup -c 11
+    - name: Install native dependencies
+      if: ${{ matrix.pytest_mark == 'not iotlab_creds' }}
+      run: |
+        sudo apt-get update
+        sudo apt-get install lib32asan5
```
Comment (kaspar030) [inline .github/workflows/release-test.yml:123]: waitaminute, does this not need a "--yes" or similar?

**Review by kaspar030 — CHANGES_REQUESTED** [review]

File: .github/workflows/release-test.yml (hunk 1 above)
Comment (kaspar030) [reply .github/workflows/release-test.yml:123]:
```suggestion
        sudo apt-get -y install lib32asan5
```

Comment (miri64) [reply .github/workflows/release-test.yml:123]: Not for Github Actions afaik. Never used it at least and it is not listed [in the doc](https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners). Also, as you can see in https://github.com/miri64/RIOT/runs/6552855739?check_suite_focus=true, it runs just fine without it ;-).

**Comment by miri64** [comment]
Piggy-backed a doc update for that new dependency.

File: .github/workflows/release-test.yml (hunk 1 above)
Comment (kaspar030) [reply .github/workflows/release-test.yml:123]: ok :)

Comment (miri64) [reply .github/workflows/release-test.yml:123]: I guess someone at Github anticipated this common tripwire ;-).

**Pull request merged**